image.go

```diff
@@ -20,10 +20,12 @@ type Image interface {
 	Target() ocispec.Descriptor
 	// Unpack unpacks the image's content into a snapshot
 	Unpack(context.Context, string) error
-	// RootFS returns the image digests
+	// RootFS returns the unpacked diffids that make up images rootfs.
 	RootFS(ctx context.Context) ([]digest.Digest, error)
-	// Size returns the image size
+	// Size returns the total size of the image's packed resources.
 	Size(ctx context.Context) (int64, error)
+	// Config returns the image config digest and image config
+	Config(ctx context.Context) (digest.Digest, *ocispec.ImageConfig, error)
```

Comment:
This overly couples the config with the oci format. Callers should simply fetch and deserialize the result from the content store.

Reply:
I could return ocispec.Image if you think that's better. :p

Reply:
Neither are correct: this assumes that the image is OCI.

Reply:
Hm, that is true.